frontend/include/chpl/uast/AttributeGroup.h

```diff
@@ -85,17 +85,17 @@ class Attributes final : public AstNode {
   void dumpInner(const DumpSettings& s) const;
 
  public:
-  ~Attributes() override = default;
+  ~AttributeGroup() override = default;
 
-  static owned<Attributes> build(Builder* builder, Location loc,
+  static owned<AttributeGroup> build(Builder* builder, Location loc,
                                  std::set<PragmaTag> pragmas,
                                  bool isDeprecated,
                                  bool isUnstable,
                                  UniqueString deprecationMessage,
                                  UniqueString unstableMessage);
```

Comment:
the indentation changed, to go back to "\~proper style\~" we'd need to indent `paragmas` etc.